features/step_definitions/sample_file_steps.rb

```diff
@@ -53,6 +53,10 @@ def a
   contents = <<-EOS.strip_heredoc
     # clean class for testing purposes
     class Clean
+      def initialize(sub)
+        @sub = sub
+      end
+
```

Comment:
Should I move the path's constants definition to a different file and require them in the `spec_helper` and also the `sample_file_steps`?